File: latihan3/python/processor.py
class processor():
    # atribut
    __name = "blank"
    __price = 0

    # constructor
    def __init__(self):
        self.__name = "blank"
        self.__price = 0

        
    def __init__(self, name="blank", price=0):
        self.__name = name;
        self.__price = price;

    # setter dan getter

    def setName(self, name):
        self.__name = name
        
    def getName(self):
        return self.__name 

    def setPrice(self, price):
        self.__price = price
        
    def getPrice(self):
        return self.__price 

File: latihan3/cpp/processor.cpp
class processor{
    private:
        // atribut
        string name;
        int price;

    public:
        // konstruktor
        processor(){}
        
        processor(string n, int p){
            this->name = n;
            this->price = p;
        }

        // setter dan getter
        void setNameP(string name){ this->name = name; }
        string getNameP(){ return this->name; }

        void setPriceP(int price){ this->price = price; }
        int getPriceP(){ return this->price; }

        // destructor
        ~processor(){}
};